connector/avro/src/main/scala/org/apache/spark/sql/avro/SchemaConverters.scala

```diff
@@ -49,13 +49,13 @@ object SchemaConverters {
   /**
    * Converts an Avro schema to a corresponding Spark SQL schema.
    *
-   * @since 2.4.0
+   * @since 4.0.0
    */
-  def toSqlType(avroSchema: Schema): SchemaType = {
```

Comment:
actually this is a public API and we can't simply remove it. Shall we add it back and make it just call `toSqlType(avroSchema, false)`?

Reply:
sure!